packages/fiori/src/ShellBar.hbs

```diff
@@ -34,42 +33,18 @@
 
 	<div class="ui5-shellbar-overflow-container ui5-shellbar-overflow-container-middle">
 		{{#if showCoPilot}}
-			{{> coPilot}}
+			<div class="ui5-shellbar-copilot-wrapper"
```

Comment:
When this is focused, the focus is blue and only seen on the sides, not on top/bottom. Is this expected?